code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -0,0 +1,826 @@
+						targets3 += H
+						// unless we hate their guts
+				if (IsInfected(H)) // target them if they attack us
+					if (H in enemies)
+						targets3 += H
+				else if (H.reagents.has_reagent("terror_black_toxin") && istype(src, /mob/living/simple_animal/hostile/poison/terror_spider/black))
+					if (get_dist(src,H) <= 2)
+						// if they come to us...
+						targets2 += H
+					else if ((H in enemies) && !H.reagents.has_reagent("terror_black_toxin",31))
+						// if we're aggressive, and they're not going to die quickly...
+						targets2 += H
+					else
+						// they are far away, and either we're not very aggressive, or they are dying already
+						// either way, not much point in targeting them
+						// if they shoot us, of course, then we will consider them a valid target
+				else
+					if (H.can_inject(null,0,"chest",0))
+						targets1 += H
+					else if (H in enemies)
+						targets2 += H
+					else
+						targets3 += H
+					// first, go after targets we can inject - lightly armored humans.
+					// second, go after humans that attacked us.
+					// third, go after other humans.
+			else if (istype(H, /mob/living/simple_animal))
+				var/mob/living/simple_animal/hostile/poison/terror_spider/M = H
+				if (M.force_threshold > melee_damage_upper)
+					// If it has such high armor it can ignore any attack we make on it, ignore it.
+				else if (M in enemies)
+					targets2 += M
+				else
+					targets3 += M
+			else
+				if (H in enemies)
+					targets2 += H
+				else
+					targets3 += H
+		for(var/obj/mecha/M in mechas_list)
+			if(get_dist(M, src) <= vision_range && can_see(src, M, vision_range))
+				if(get_dist(M, src) <= 2)
+					targets2 += M
+				else
+					targets3 += M
+		if (health < maxHealth)
+			// very unlikely that we're being shot at by a space pod - so only check for this if our health is lower than max.
+			for(var/obj/spacepod/S in spacepods_list)
+				if(get_dist(S, src) <= vision_range && can_see(src, S, vision_range))
+					targets3 += S
+			// Turret-targeting code is broken. Also such a rare case. We leave it commented for now.
+			/*
+			for(var/obj/machinery/porta_turret/R in view(src, vision_range))
+				if (!R.stat)
+					// spiders will target turrets that shoot them, by default
+					targets3 += R
+			*/
+		if (targets1.len)
+			return targets1
+		else if (targets2.len)
+			return targets2
+		else
+			return targets3
+	else if (ai_type == 1)
+		// DEFEND SELF ONLY
+		//var/list/Mobs = hearers(vision_range, src) - src
+		for(var/mob/living/H in view(src, vision_range))
+		//for(var/mob/H in Mobs)
+			if (H.stat == 2)
+				// dead mobs are ALWAYS ignored.
+			else if (!stat_attack && H.stat == 1)
+				// unconscious mobs are ignored unless spider has stat_attack
+			else if (H in enemies)
+				targets1 += H
+		for(var/obj/mecha/M in mechas_list)
+			if(M in enemies && get_dist(M, src) <= vision_range && can_see(src, M, vision_range))
+				targets1 += M
+		for(var/obj/spacepod/S in spacepods_list)
```

Comment:
https://github.com/ParadiseSS13/Paradise/blob/64dbdc42db5a6f7f5fb674e39d7463cf5979a253/code/modules/mob/living/simple_animal/hostile/hostile.dm at line 72, again. This is standard for all simple_animals. Y'all are giving me flak for doing literally the same thing that every other simple_animal mob in the game does.